src/azure-cli/azure/cli/command_modules/backup/tests/latest/recordings/test_backup_archive.yaml

```diff
@@ -514,7 +514,7 @@ interactions:
       User-Agent:
       - AZURECLI/2.60.0 azsdk-python-core/1.30.1 Python/3.10.11 (Windows-10-10.0.22631-SP0)
     method: GET
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Compute/locations/eastus2euap/publishers/MicrosoftWindowsServer/artifacttypes/vmimage/offers/WindowsServer/skus/2012-R2-Datacenter/versions?$top=1&$orderby=name%20desc&api-version=2024-07-01
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Compute/locations/eastus2euap/publishers/MicrosoftWindowsServer/artifacttypes/vmimage/offers/WindowsServer/skus/2012-R2-Datacenter/versions?$top=1&$orderby=name%20desc&api-version=2024-11-01
```

Comment:
May I ask if these tests directly replace the api-version without performing live testing to re-record the YAML file? If so, could you please tell me the reason?

Reply:
Yes,due to service issues, replace the api-version